x-pack/plugins/security_solution/public/management/components/paginated_content/paginated_content.tsx

```diff
@@ -164,9 +164,19 @@ export const PaginatedContent = memo(
 
     const handleItemsPerPageChange: EuiTablePaginationProps['onChangeItemsPerPage'] = useCallback(
       (pageSize) => {
-        onChange({ pageSize, pageIndex: pagination?.pageIndex || 0 });
+        if (pagination?.pageIndex) {
+          const pageIndex = Math.floor(
+            ((pagination?.pageIndex ?? 0) * (pagination?.pageSize ?? 10)) / pageSize
```

Comment:
we have constants with these defaults values inside management/common/constants.ts

```
export const MANAGEMENT_PAGE_SIZE_OPTIONS: readonly number[] = [10, 20, 50];
export const MANAGEMENT_DEFAULT_PAGE = 0;
export const MANAGEMENT_DEFAULT_PAGE_SIZE = 10;
```